src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthConfiguration.java

```diff
@@ -19,16 +19,18 @@
 
 public class OAuthConfiguration extends Configuration
 {
-    public static final PropertyDef<String> AUTH_JWT_KEYS;;
+    public static final PropertyDef<String> AUTH_JWT_KEYS;
 
     private static final ConfigurationDef AUTH_JWT_CONFIG;
+    public static final BooleanPropertyDef TOKEN_EXPIRATION_CHECK;
```

Comment:
Please make constant `private` scope.
Please rename `TOKEN_EXPIRATION_CHECK` to `AUTO_EXPIRE_IN_FLIGHT_REQUESTS` with default `true`.